qiita_pet/handlers/base_handlers.py

```diff
@@ -59,7 +56,12 @@ def write_error(self, status_code, **kwargs):
         request_info = ''.join(["<strong>%s</strong>: %s\n" %
                                (k, req_dict[k]) for k in
                                 req_dict.keys() if k != 'files'])
-        error = exc_info[1]
+        error = str(exc_info[1]).split(':', 1)[1]
```

Comment:
Just to confirm, these array elements, the split on ':', etc. will work in all cases?

Reply:
All cases I have tested ... cause the error gets prepended: `Error X: bla`.